includes/payment-methods/class-giropay-payment-method.php

```diff
@@ -26,7 +27,7 @@ public function __construct( $token_service ) {
 		$this->stripe_id   = self::PAYMENT_METHOD_STRIPE_ID;
 		$this->title       = 'giropay';
 		$this->is_reusable = false;
-		$this->currencies  = [ 'EUR' ];
+		$this->currencies  = [ Currency_Code::EUROZONE ];
```

Comment:
Just curious, why `EUROZONE` and not just `EURO`?

Reply:
Because I felt it was consistent with naming other countries. `EURO` is a currency, `EUROZONE` is a group.